main.cpp

```diff
@@ -190,14 +375,41 @@ struct Screen
     double boarderPerimeterWidth = 2.4;
     //touch sensitivity
     double touchSensitivity = 2.3;
+
+    //init screen
+    double currentPosition = 0;
+    double currentSpeed = 0;
+    bool clicking = false;
+
     // 3 things it can do:
     //1) Scroll 
     void screenScroll(double screenPosition, double scrollSpeed);
     //2) click via touch
-    bool touchClick (bool pressed);
+    void touchClick (bool pressed);
     //3) control fx via touchbars sending midi cc's
-    void controlCC(std::string midiCCNum, int ccData);
+    std::string controlCC(std::string midiCCNum);
 };
+
+void Screen::screenScroll(double screenPosition, double scrollSpeed)
+{
+    currentPosition = screenPosition;
+    currentSpeed = scrollSpeed;
+}
+void Screen::touchClick (bool pressed)
+{
+    if (pressed == true)
+    {
+        clicking = true;
+    } else 
+    {
+        clicking = false;
+    }
+}
```

Comment:
Curly braces go on their own lines
you can just write `clicking = pressed;` and skip the `if/else` altogether.
- [ ] revise